pmm-app/src/pmm-qan/panel/components/Details/Table/TableContainer.hooks.ts

```diff
@@ -6,9 +6,12 @@ import { Databases } from '../Details.types';
 export const useTables = (examples, databaseType): any[] => {
   const [jsonExplain, classicExplain] = useExplains(examples, databaseType);
   const [tables, setTables] = useState<any[]>([]);
+  const [loading, setLoading] = useState<boolean>(false);
 
   useEffect(() => {
     const getTables = async () => {
+      setLoading(true);
```

Comment:
I wonder, are we fixing something else in this PR, not only fontawesome?

Reply:
Beata found an issue with a spinner that wasn't completely fixed in another task and in my opinion, it was the best option to put this small fix right here because it's related to spinner